TYPO3.Neos/Classes/TYPO3/Neos/Controller/Module/Management/AssetController.php

```diff
@@ -89,6 +89,18 @@ class AssetController extends \TYPO3\Media\Controller\AssetController
     protected $userService;
 
     /**
+     * @Flow\Inject
+     * @var DomainUserService
+     */
+    protected $domainUserService;
+
+    /**
+     * @var ContentDimensionPresetSourceInterface
+     * @Flow\Inject
```

Comment:
All other class properties define first `@Flow\Inject` and then `@var`. Should be snyced